iroh-resolver/src/unixfs_builder.rs

```diff
@@ -638,6 +643,47 @@ mod tests {
         Ok(())
     }
 
+    // read an AsyncRead into a vec completely
+    async fn read_to_vec<T: AsyncRead + Unpin>(mut reader: T) -> Vec<u8> {
+        let mut out = Vec::new();
+        reader.read_to_end(&mut out).await.unwrap();
+        out
+    }
+
+    /// Read a stream of (cid, block) pairs into an in memory store and return the store and the root cid
+    async fn stream_to_resolver(
+        stream: impl Stream<Item = Result<(Cid, Bytes)>>,
+    ) -> Result<(Cid, Resolver<HashMap<Cid, Bytes>>)> {
+        tokio::pin!(stream);
+        let items: Vec<_> = stream.try_collect().await?;
+        let (root, _) = items.last().context("no root")?.clone();
+        let store: HashMap<Cid, Bytes> = items.into_iter().collect();
+        let resolver = Resolver::new(store);
+        Ok((root, resolver))
+    }
+
+    #[ignore]
```

Comment:
Can you add a comment that this is ignored, because it fails please?

Reply:
OK